0-email-signup-form/FCCvote-net-neutrality-thanks/above-form-content.html

```diff
@@ -26,6 +26,9 @@
                   <input aria-label="Email Address" placeholder="youremail@example.com" id="email" name="email" type="email" required="" data-parsley-trigger="change">
                 </div>
               </div>
+              <div class="field-container full">
+                <input type="text" name="country" value="US" class="hidden" required="" readonly>
+              </div>
```

Comment:
Again, I can't recall the reason why `type="hidden"` can't be used.  Let's just use `class=hidden`